tasks/kmt.py

```diff
@@ -617,6 +687,139 @@ def build_run_config(run: str | None, packages: list[str]):
     return c
 
 
+def build_target_packages(filter_packages):
+    all_packages = go_package_dirs(TEST_PACKAGES_LIST, [NPM_TAG, BPF_TAG])
+    if filter_packages == []:
+        return all_packages
+
+    filtered = list()
+    for pkg in all_packages:
+        if os.path.relpath(pkg) in filter_packages:
+            filtered.append(pkg)
+
+    return filtered
+
+
+def build_object_files(ctx, fp):
+    ninja_generate(ctx, fp)
+    ctx.run(f"ninja -d explain -f {fp}")
+
+
+@task
+def kmt_sysprobe_prepare(
+    ctx: Context,
+    arch: ArchOrLocal,
+    stack: str | None = None,
+    kernel_release: str | None = None,
+    packages=None,
+    extra_arguments: str | None = None,
+    ci: bool = False,
+):
+    if ci:
+        stack = "ci"
+
+    if stack is None:
+        raise Exit("A stack name must be provided")
+
+    if arch is None or arch == "local":
+        raise Exit("No architecture provided")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Could be replaced by asserts